subsys/usb/device_next/class/usbd_uac2_macros.h

```diff
@@ -32,6 +33,33 @@
 #define IS_ARRAY_SORTED(node, prop)						\
 	DT_FOREACH_PROP_ELEM_SEP(node, prop, ARRAY_ELEMENT_LESS_THAN_NEXT, (&&))
 
+#define FIRST_INTERFACE_NUMBER			0x00
+#define FIRST_IN_EP_ADDR			0x81
+#define FIRST_OUT_EP_ADDR			0x01
+
+/* A.1 Audio Function Class Code */
+#define AUDIO_FUNCTION				AUDIO
+
+/* A.2 Audio Function Subclass Codes */
+#define FUNCTION_SUBCLASS_UNDEFINED		0x00
+
+/* A.3 Audio Function Protocol Codes */
+#define FUNCTION_PROTOCOL_UNDEFINED		0x00
+#define AF_VERSION_02_00			IP_VERSION_02_00
+
+/* A.4 Audio Interface Class Code */
+#define AUDIO					0x01
+
+/* A.5 Audio Interface Subclass Codes */
+#define INTERFACE_SUBCLASS_UNDEFINED		0x00
+#define AUDIOCONTROL				0x01
+#define AUDIOSTREAMING				0x02
+#define MIDISTREAMING				0x03
```

Comment:
Just a note, we also have them in include/zephyr/usb/class/usb_audio.h.

Reply:
The old usb public usb audio header is not useful at all for the new implementation. The two versions of the spec differ too much. The USB Audio 2 implementation is targeted towards the device-next stack and will use new application API.

Moreover, AUDIOCONTROL, AUDIOSTREAMING, MIDISTREAMING are really implementation details that should not be part of public API at all. I prefer to just leave the `include/zephyr/usb/class/usb_audio.h` intact as it'll be removed anyway after we deprecate and then remove the old USB stack.